src/components/Vladilena_Form/Vladilena_Form.tsx

```diff
@@ -67,7 +70,7 @@ export function Vladilena_Form() {
           </SheetHeader>
 
           <div className="grid gap-5">
-            {fields.map(({ id, label, type, placeholder, icon }) => (
+            {fields.map(({ id, label, type, placeholder, icon, min, max }) => (
```

Comment:
Separate the inputs. It is not good ideas to keep them like that. Because if you need to change the style of one, let's say make the wright red, you would change the signature of your he function